packages/shared/src/hooks/notifications/useAcceptedPushNow.ts

```diff
@@ -0,0 +1,31 @@
+import { useCallback, useMemo } from 'react';
+import { useQuery, useQueryClient } from '@tanstack/react-query';
+import { generateQueryKey, RequestKey } from '../../lib/query';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { disabledRefetch } from '../../lib/func';
+
+interface UseAcceptedPushNow {
+  onAcceptedJustNow: (accepted: boolean) => void;
+  acceptedJustNow: boolean;
+}
+
+export const useAcceptedPushNow = (): UseAcceptedPushNow => {
```

Comment:
Just to understand why "now" is needed, on what specific flow do we need to say user just now accepted push vs accepted in general?

Reply:
We display some sort of UI but keeps the same banner. When coming back, the banner will be gone completely.

Reply:
Ah, ok, was thinking do we need to cleanup that value on page navigation or something? Since it will stay true until next full page refresh. If that was meant to happen then it is ok 👍